### PR TITLE
update react-native version

### PR DESCRIPTION
**Pull request by future-challenger**
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:

> **Unless you are a React Native release maintainer and cherry-picking an *existing* commit into a current release, ensure your pull request is targeting the `master` React Native branch.**

Explain the **motivation** for making this change. What existing problem does the pull request solve?

Prefer **small pull requests**. These are much easier to review and more likely to get merged. Make sure the PR does only one thing, otherwise please split it.

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Make sure tests pass on both Travis and Circle CI.

**Code formatting**

Look around. Match the style of the rest of the codebase. See also the simple [style guide](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#style-guide).

For more info, see the ["Pull Requests" section of our "Contributing" guidelines](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#pull-requests).

update react-native version to 0.32

### CODE REVIEW

**Comment by facebook-github-bot-8** [comment]
By analyzing the blame information on this pull request, we identified @sreesharp and @GantMan to be potential reviewers.

**Comment by facebook-github-bot-8** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by GantMan** [comment]
This actually breaks the readability.  You need to change the version in 2 places for it to make sense.

not so sure this should always be updated to the latest version of RN, that's a bit of a headache of a task, imo.

File: docs/Upgrading.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Note the latest version of the `react-native` npm package from here (or use `npm
 Now install that version of `react-native` in your project with `npm install --save`. For example, to upgrade to the version `0.31`, in a terminal run:
```
Comment (GantMan) [inline docs/Upgrading.md]: You have version 0.31 still here.

**Comment by hramos** [comment]
Agree with @GantMan, unless the release maintainer wants to make sure they update this file whenever a new release is cut.

If we don't want to use a version number that will soon become stale, we can use an obviously fake version number _or_ a placeholder string instead (e.g. npm install --save react-native@SPECIFY_VERSION_NUMBER_HERE)

**Comment by mkonicek** [comment]
Agree with @GantMan and @hramos. Let's keep it simple, hopefully people will understand to use the latest version.

**Pull request closed without merging**